gke-deploy/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:stretch AS build-env
+RUN go get github.com/gobuffalo/packr/v2/packr2
+ADD . /go/src/github.com/GoogleCloudPlatform/cloud-builders/gke-deploy
+RUN go get -d github.com/GoogleCloudPlatform/cloud-builders/gke-deploy
+RUN cd /go/src/github.com/GoogleCloudPlatform/cloud-builders/gke-deploy && packr2
+RUN go build -o /gke-deploy github.com/GoogleCloudPlatform/cloud-builders/gke-deploy
+
+FROM google/cloud-sdk:alpine
+RUN gcloud -q components install kubectl
```

Comment:
Does this tool need to shell out to `kubectl`, or can it make direct API calls to the cluster, to remove this dependency?

Reply:
Yes, we decided to using `kubectl` was the best solution for us to get this out the door.